docs/premium-features/break-the-monolith.md

```diff
@@ -40,30 +36,42 @@ Overview mode is the view-only default mode for viewing your services and entiti
 
 #### Redesign Mode
 
-Redesign mode allows you redesign one service at a time and migrate entities.
+Redesign mode allows you redesign one service at a time, migrate, and create new entities.
+
+::note
+**Free users** have access to Overview mode _without_ the Redesign capabilities.
+:::
 
 ### Step 2: Manual Service Redesign
 
-After entering Redesign Mode, you can manually move your entity between existing or new services to create miro-services.
+After entering Redesign Mode, you can manually move your entities between existing or new services to create miro-services.
 
 To migrate entities to other services you can drag and drop them into the appropriate service. After making changes, you can click on the **Apply** button to implement those changes.
 
-### Step 3: Use AI for Service Breakdown (Optional)
+<!--
+### Step 3: Utilize AI for Smart Architecture Suggestions (Optional)
 
-You can also optionally use AI to help you transition your entities and which micro-services should be created.
-
-The "AI Helper" button, also present in the Project and Service Overview screens, activates the AI-driven process that suggests the most optimal way to name your services and move them into a microservice architecture. This process requires selecting a service and reviewing the AI's recommendations.
+The "Break the Monolith" button, available in the Project and Service Overview screens, triggers AI analysis to suggest an efficient new microservice architecture, including service distribution and entity organization. Review the AI's recommendations before making any changes.
 
 :::note
 **This feature uses LLMs to analyze and suggest ideal microservices structures**. It requires sharing details of your entities and fields with LLMs for analysis. We ensure the highest privacy standards and use your data only to improve your project's architecture.
```

Comment:
IIUC- this comment was not implemented -
"Let's also say that (1) only anonymous data will be shared, (2) it will only be shared when the user pro-actively uses features that utilize LLMs, (3) the users will be able to opt-out of the LLMs features."